### PR TITLE
little change

### PR DESCRIPTION
**Pull request by dergil**

### CODE REVIEW

**Review by vincemann — COMMENTED** [review]
-

File: src/packets/SubscribePacket.h [hunk 1]
```diff
@@ -1,6 +1,7 @@
 //
 // Created by derdoc on 7/4/21.
 //
+// CHANGED
```
Comment (vincemann) [inline src/packets/SubscribePacket.h]: change to dessen gil

Comment (dergil) [reply src/packets/SubscribePacket.h]: Really ?!?!?

**Review by vincemann — APPROVED** [review]
good job gil

File: src/packets/parsers/SubscribePacketParser.cpp [hunk 2]
```diff
@@ -1,6 +1,7 @@
 //
 // Created by derdoc on 7/4/21.
 //
+// NOT APPROVED
```
Comment (vincemann) [inline src/packets/parsers/SubscribePacketParser.cpp]: Remove this comment

File: src/packets/parsers/SubscribePacketParser.h [hunk 3]
```diff
@@ -1,6 +1,7 @@
 //
 // Created by derdoc on 7/4/21.
 //
+// APPROVED
```
Comment (vincemann) [inline src/packets/parsers/SubscribePacketParser.h]: Change this to Approved V2

File: src/packets/parsers/SubscribePacketParser.cpp [hunk 4]
```diff
@@ -1,6 +1,7 @@
 //
 // Created by derdoc on 7/4/21.
 //
```
Comment (dergil) [inline src/packets/parsers/SubscribePacketParser.cpp]: my very own comment

Comment (dergil) [reply src/packets/parsers/SubscribePacketParser.cpp]: my reply

**Review by vincemann — CHANGES_REQUESTED** [review]

File: src/packets/parsers/SubscribePacketParser.cpp (hunk 2 above)
Comment (dergil) [reply src/packets/parsers/SubscribePacketParser.cpp]: test

File: src/packets/parsers/SubscribePacketParser.cpp [hunk 5]
```diff
@@ -1,7 +1,6 @@
...
-// NOT APPROVED
```
Comment (vincemann) [inline src/packets/parsers/SubscribePacketParser.cpp]: correct

File: src/packets/parsers/SubscribePacketParser.h [hunk 6]
```diff
@@ -1,7 +1,7 @@
 //
 // Created by derdoc on 7/4/21.
 //
-// APPROVED
+// APPROVED V2
```
Comment (vincemann) [inline src/packets/parsers/SubscribePacketParser.h:4]: perfect

**Review by vincemann — APPROVED** [review]

**Pull request merged**